### PR TITLE
Added files for setup to resolve #165

### LINKED ISSUE #165: Set up setup.py and wheel configuration
For eventual PyPi release.

### PR DESCRIPTION
**Pull request by captainsafia**
@rgbkrk — take a look. Let me know if this covers everything you showed me on live chat.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+from distutils.core import setup
+import sys
+
+if "develop" in sys.argv or any(arg.startswith("bdist") for arg in sys.argv):
+    import setuptools
+
+setup_args = dict(
+    name="tmpnb",
+    version="0.1.0",
+    description="Tool for aunching temporary Jupyter notebook servers",
```
Comment (rgbkrk) [inline setup.py]: s/aunching/launching

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    url="https://github.com/jupyter/tmpnb",
+    license="BSD",
+    author="Jupyter Development Team",
+    author_email="jupyter@googlegroups.com",
+    platforms="Linux, Mac OS X"
```
Comment (rgbkrk) [inline setup.py]: :+1:

**Comment by captainsafia** [comment]
@rgbkrk — MERGE ME! :D

**Comment by minrk** [comment]
@captainsafia it's awesome that you cut 0.1 :tada:! In the future, can you make tags that are just the version number, without 'v' prefix?

**Comment by rgbkrk** [comment]
> can you make tags that are just the version number, without 'v' prefix

That may be my fault, I probably suggested looking at one of our nodejs packages instead of a python package. :fearful:

Is this to make it match the format in setup.py (e.g. `version='0.1.0'`)?

**Comment by rgbkrk** [comment]
We can go ahead and change the tag after the fact I assume. We should also pin that release on Docker Hub too, so we have `jupyter/tmpnb:0.1.0`.

**Comment by minrk** [comment]
> We can go ahead and change the tag after the fact I assume

Yeah, we can. I pushed the same commit as 0.1.0 already instead of taking down v0.1.0, but we can take it down and move the release info over, if we want. It isn't a big deal.

**Comment by rgbkrk** [comment]
I was about to move the release info over, but I'll leave that for you @captainsafia so you get the credit. Draft a new release using the new tag, copy your text over. :+1:

**Comment by minrk** [comment]
:+1:

**Comment by captainsafia** [comment]
> I'll leave that for you @captainsafia so you get the credit. 

No credit needed! Just cat gfis. I shall execute release surgery now.

**Comment by captainsafia** [comment]
Release surgery complete. Hopefully, I don't have to use my malpractice insurance.

**Pull request merged**